language-support/ts/codegen/tests/ts/generated/src/__tests__/test.ts

```diff
@@ -110,7 +110,8 @@ test('create + fetch & exercise', async () => {
     tuple: {_1: '12', _2: 'mmm'},
     enum: Main.Color.Red,
     enumList: [Main.Color.Red, Main.Color.Blue, Main.Color.Yellow],
-    variant: {tag: 'Add', value: {_1:{tag: 'Lit', value: '1'}, _2:{tag: 'Lit', value: '2'}}}
+    variant: {tag: 'Add', value: {_1:{tag: 'Lit', value: '1'}, _2:{tag: 'Lit', value: '2'}}},
+    sumProd: {tag: 'Corge', value: {x:'1', y:'Garlpy'}}
```

Comment:
```suggestion
    sumProd: {tag: 'Corge', value: {x:'1', y:'Garlpy'}},
```
Let's add the trailing comma so we don't always get a two-line change whenever we want to add a single line.